gui/templates/storage/index.html

```diff
@@ -14,6 +14,13 @@
   refreshOnShow: true" class="objrefresh data_storage_Task" tab="storage.Task.View" id="tab_Task">
   </div>
 
+  <div data-dojo-type="dijit.layout.ContentPane" data-dojo-props="title: '{% trans "Quota Alerts"|escapejs %}'
+  {% if focused_tab == "storage.RootDataset.View" %}, selected: true{% endif %},
+  doLayout: false,
+  href: '{{ rootdataset_edit_url }}',
+  refreshOnShow: true" class="objrefresh data_storage_RootDataset" tab="storage.RootDataset.View" id="tab_RootDataset">
+  </div>
```

Comment:
I dont get the concept of RootDataset. Why do we need something exclusive for it?

Reply:
Where would we store default values to use when dataset's value is absent or inherited?

Reply:
I dont see an advantage of having configurable default values if we have a way to customize them per-dataset, specially with inheritable ZFS user properties.

That being said I think its fine to hard code defaults for the sake of keeping things simple, or I am missing more fundamental?